### PR TITLE
Lit6370 incarnation

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**
I've just found it by accident, I am afraid that it might be something problematic because https://betamasaheft.eu/works/LIT6370Incarnation/main .

### CODE REVIEW

File: new/LIT6370Incarnation.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6370Incarnation" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ድርሳን፡ በእንተ፡ ተሠግዎቱ፡ ለእግዚአብሔር፡ ቃል፡</title>
+                <title xml:lang="gez" corresp="#t1">Dǝrsān baʾǝnta taśagǝwotu la-ʾƎgziʾabǝḥer qāl</title>
```
Comment (thea-m) [inline new/LIT6370Incarnation.xml:9]: taśśagǝwotu ?

**Review by thea-m — CHANGES_REQUESTED** [review]

**Comment by thea-m** [comment]
I'm not sure about the error message - the record couldn't be displayed while its still unmerged. Let's wait until the merge and then check again.

File: new/LIT6370Incarnation.xml (hunk 1 above)
Comment (MarcinKrawczuk) [reply new/LIT6370Incarnation.xml:9]: Leslau gives one ś

Comment (thea-m) [reply new/LIT6370Incarnation.xml:9]: I meant two g 🙈

**Review by thea-m — APPROVED** [review]

**Pull request merged**